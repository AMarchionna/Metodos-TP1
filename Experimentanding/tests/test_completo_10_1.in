10 45
1 1 1 2 0
1 1 0 3 1
1 1 1 4 0
1 1 0 5 1
1 1 0 6 1
1 1 1 7 0
1 1 1 8 0
1 1 1 9 0
1 1 0 10 1
1 2 1 3 0
1 2 1 4 0
1 2 0 5 1
1 2 1 6 0
1 2 1 7 0
1 2 0 8 1
1 2 0 9 1
1 2 0 10 1
1 3 0 4 1
1 3 0 5 1
1 3 0 6 1
1 3 1 7 0
1 3 0 8 1
1 3 0 9 1
1 3 1 10 0
1 4 1 5 0
1 4 1 6 0
1 4 0 7 1
1 4 1 8 0
1 4 1 9 0
1 4 0 10 1
1 5 0 6 1
1 5 1 7 0
1 5 0 8 1
1 5 0 9 1
1 5 0 10 1
1 6 0 7 1
1 6 0 8 1
1 6 0 9 1
1 6 0 10 1
1 7 1 8 0
1 7 1 9 0
1 7 0 10 1
1 8 1 9 0
1 8 0 10 1
1 9 0 10 1
