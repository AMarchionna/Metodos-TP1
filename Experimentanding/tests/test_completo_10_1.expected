0.54166666952666
0.45833334405783
0.37500001076589
0.54166666373867
0.45833333463509
0.37500000879669
0.45833333343768
0.54166665980679
0.45833333660159
0.79166667295309